index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <main>
+   <header>
+    <div class="oluchi"><a href="#">Oluchi Theresa</a></div>
+    <div class="port"><a href="#">Portfolio</a></div>
+    <div class="about"><a href="#">About</a></div>
+    <div class="contact"><a href="#">Contact</a></div>
+   </header>
+   <div class="container">
+    <section>
```

Comment:
- [ ] I think it would be better to use the `<section>` tag instead of  using `<div>` tag 👌 for example `<section class="container">`, Then you can remove line 19 💯